x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/StatementParserTests.java

```diff
@@ -962,6 +964,16 @@ public void testInputParams() {
         assertThat(alias.child().fold(), is(11));
     }
 
+    public void testMatch() throws IOException {
+        String queryString = "field: value";
+        assertEquals(
+            new Filter(EMPTY, PROCESSING_CMD_INPUT, new StringQueryPredicate(EMPTY, queryString, null)),
+            processingCommand("match \"" + queryString + "\"")
+        );
+
+        expectError("from a | match an unquoted string", "mismatched input 'an' expecting QUOTED_STRING");
```

Comment:
This might need an `assumeFalse(Build.current().isSnapshot()` or it'll fail the "release tests".

Reply:
Makes sense, done in 511282983155d1e11b9acb59c171ae6bfd25fa25